npm/npm.go

```diff
@@ -106,6 +155,7 @@ func GetAIMetadata() string {
 }
 
 // SendClusterMetrics :- send NPM cluster metrics using AppInsights
+// QUESTION(jungukcho): Where is this function called?
```

Comment:
This function is important for us to send periodic metrics about cluster size to Prometheus. Looks like it got removed in one of the previous PRs which i think is a mistake and we should be calling a go routine on this for periodic updates.

https://github.com/Azure/azure-container-networking/commit/d1699290486d3b2faa7e79bbae7aa5bca2830857#diff-22ef58b44b664ba24dd77fa2feeb41115c5b9b290f7406c4c642c38ba44cc9feL78

Reply:
These metrics are being sent back to our local AITelemetry to log information on the cluster. I think we can safely remove these functions.

Reply:
Good catch.  
I think it is good to keep those numbers. Interval is 30min and I don't think it is heavy operations, but we can get useful metrics. I will add the removed code again. 
Let me know  if you have another opinion.

Later, we need to consolidate all metrics-related codes in metric packages in a nice way. 
Let's put it as future item.